server/game/cards/03-DotV/IsawaTadaka.js

```diff
@@ -5,8 +5,7 @@ class IsawaTadaka extends DrawCard {
     setupCardAbilities(ability) {
         this.persistentEffect({
             targetController: Players.Opponent,
-            condition: context => context.game.currentConflict && context.game.currentConflict.hasElement('earth') ||
-                context.game.rings.earth.isConsideredClaimed(context.player),
+            condition: context => context.game.rings.earth.isConsideredClaimed(context.player.opponent),
```

Comment:
Shouldn't this be "!context.game.rings.earth.isConsideredClaimed(context.player.opponent)"?

Reply:
Yes, it should